src/constants.ts

```diff
@@ -41,3 +41,13 @@ export enum PermissionAction {
   TENANT_SQL_UPDATE = 'tenant:Sql:Write:Update',
   UPDATE = 'write:Update',
 }
+
+export enum SupportCategories {
+  PROBLEM = 'problem',
```

Comment:
I think you'll need to retain the existing sentence casing here-we can't really synchronize rollout of API and frontend, so the first system to upgrade will cause a break until the second system comes into sync.

Reply:
gotcha! pushed an update